ui/app/ducks/gas/gas-duck.test.js

```diff
@@ -321,10 +321,10 @@ describe('Gas Duck', () => {
       assert.deepEqual(
         global.fetch.getCall(0).args,
         [
-          'https://dev.blockscale.net/api/gasexpress.json',
+          'https://ethgasstation.info/json/ethgasAPI.json',
           {
             'headers': {},
-            'referrer': 'https://dev.blockscale.net/api/',
+            'referrer': 'http://ethgasstation.info/json/',
```

Comment:
I'm curious to know why `http` is used here instead of `https`. I noticed other fetches to ethgasstation set this referrer as well.